### ISSUE TITLE
训练完成，没有报错，但LORA在推理时没有生效。

### ISSUE THREAD

**Issue by su400** [None9527/None_Z-image-Turbo_trainer#5]
{
  "name": "lll",
  "model_type": "zimage",
  "acrf": {
    "enable_turbo": true,
    "turbo_steps": 10,
    "shift": 3,
    "jitter_scale": 0.02,
    "use_dynamic_shifting": true,
    "base_shift": 0.5,
    "max_shift": 1.15,
    "snr_gamma": 5,
    "snr_floor": 0.1,
    "use_anchor": true,
    "raft_mode": false,
    "free_stream_ratio": 0.3,
    "l2_schedule_mode": "constant",
    "l2_initial_ratio": 0.3,
    "l2_final_ratio": 0.3,
    "l2_milestones": "",
    "l2_include_anchor": false,
    "l2_anchor_ratio": 0.3,
    "latent_jitter_scale": 0
  },
  "network": {
    "dim": 16,
    "alpha": 16
  },
  "lora": {
    "resume_training": false,
    "resume_lora_path": "",
    "train_adaln": false,
    "train_norm": false,
    "train_single_stream": false
  },
  "optimizer": {
    "type": "AdamW8bit",
    "learning_rate": "1e-4"
  },
  "training": {
    "output_name": "lll",
    "learning_rate": 5e-05,
    "learning_rate_str": "5e-5",
    "weight_decay": 0,
    "lr_scheduler": "constant",
    "lr_warmup_steps": 0,
    "lr_num_cycles": 1,
    "lambda_l1": 1,
    "lambda_cosine": 0.1,
    "enable_freq": false,
    "lambda_freq": 0.3,
    "alpha_hf": 1,
    "beta_lf": 0.2,
    "enable_style": false,
    "lambda_style": 0.3,
    "lambda_struct": 1,
    "lambda_light": 0.5,
    "lambda_color": 0.3,
    "lambda_tex": 0.5,
    "lambda_fft": 0
  },
  "dataset": {
    "batch_size": 1,
    "shuffle": true,
    "enable_bucket": true,
    "datasets": [
      {
        "cache_directory": "/home/kkk/ai/lora-scripts-v1.8.5/train/lll/8_gll",
        "num_repeats": 20,
        "resolution_limit": 1536
      }
    ]
  },
  "reg_dataset": {
    "enabled": false,
    "weight": 1,
    "ratio": 0.5,
    "datasets": []
  },
  "advanced": {
    "max_grad_norm": 1,
    "gradient_checkpointing": true,
    "blocks_to_swap": 0,
    "num_train_epochs": 10,
    "save_every_n_epochs": 1,
    "gradient_accumulation_steps": 4,
    "mixed_precision": "bf16",
    "seed": 42,
    "num_gpus": 8,
    "gpu_ids": "0,1,2,3,4,5,6,7"
  },
  "modified": "2025-12-13T11:12:53.886799",
  "created": "2025-12-12T16:44:24.681599"
}                                                    这个配置有问题吗？

**Comment by su400** [comment]
我最多一次是学习50次，10轮，也是没有效果，我更新代码试一下，非常感谢，Ui还是很好用的。

**Comment by None9527** [comment]
> 我最多一次是学习50次，10轮，也是没有效果，我更新代码试一下，非常感谢，Ui还是很好用的。

这个模型训练和其它模型思路不太一样 我也踩了很多坑 有需要可以加我沟通 后续我会录一个视频稍微讲解一下
